src/MSONOneOfParser.h

```diff
@@ -0,0 +1,80 @@
+//
+//  MSONOneOfParser.h
+//  snowcrash
+//
+//  Created by Pavan Kumar Sunkara on 11/5/14.
+//  Copyright (c) 2014 Apiary Inc. All rights reserved.
+//
+
+#ifndef SNOWCRASH_MSONONEOFPARSER_H
+#define SNOWCRASH_MSONONEOFPARSER_H
+
+#include "MSONMixinParser.h"
+
+using namespace scpl;
+
+namespace snowcrash {
+
+    /** MSON One Of matching regex */
+    const char* const MSONOneOfRegex = "^[[:blank:]]*[Oo]ne [Oo]f[[:blank:]]*$";
```

Comment:
The white space should tolerate more than just one space (e.g. any white space tabs or an arbitrary number of spaces)